plugins/Live/VisitorDetails.php

```diff
@@ -309,4 +308,24 @@ public static function getReferrerSummaryForVisit($visit)
 
         return $visit->getColumn('referrerName');
     }
+
+    /**
+     * @return array
+    */
+    private function getAdditionalUrlsForSite(): array
+    {
+        $cache = Cache::getTransientCache();
```

Comment:
@mneudert fyi the transient cache is still a bit slow and this might be called a lot. Because the system is already caching all visitorDetails instances, we may be able to simplify this and could maybe use a private property like `$this->siteUrlsCached[$idSite] = ...`.

It would make caching of these things faster and simpler. 

The instances should be created only once and cached here: https://github.com/matomo-org/matomo/blob/5.0.2/plugins/Live/Visitor.php#L50-L103 
Might be good to double check though

Reply:
Will check and adjust :eyes:

Reply:
Also only found that one place for instantiation, so skipping the cache in favor of a private variable :+1: